### PR TITLE
projects/plasmasturm.org

### PR DESCRIPTION
**Pull request by magnusviri**
This doesn't have a release or tag so I hardcoded the version.

### CODE REVIEW

File: projects/plasmasturm.org/rename/package.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+build:
+  script: |
+    pod2man rename rename.1
+    mkdir -p "{{ prefix }}"/bin
+    mv rename "{{ prefix }}"/bin
+    mkdir -p "{{ prefix }}"/share/man/man1
+    mv rename.1 "{{ prefix }}"/share/man/man1
+  env:
+    ARGS:
+      - --prefix="{{prefix}}"
```
Comment (mxcl) [inline projects/plasmasturm.org/rename/package.yml]: unused

File: projects/plasmasturm.org/rename/package.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+distributable:
+  url: https://github.com/ap/rename/archive/refs/heads/master.tar.gz
```
Comment (mxcl) [inline projects/plasmasturm.org/rename/package.yml]: Should be: https://github.com/ap/rename/archive/0da9faa241093c90a40b15a9ff4b36e9858d4516.tar.gz IMO

**Review by mxcl — CHANGES_REQUESTED** [review]

File: projects/plasmasturm.org/rename/package.yml (hunk 1 above)
Comment (magnusviri) [reply projects/plasmasturm.org/rename/package.yml]: I removed the unused env var

File: projects/plasmasturm.org/rename/package.yml (hunk 2 above)
Comment (magnusviri) [reply projects/plasmasturm.org/rename/package.yml]: Turns out the developer was using tags, I just didn't notice. I switched it to use tags.

**Pull request merged**